packages/genesis/hardhat.config.ts

```diff
@@ -115,7 +115,7 @@ task('abi', 'Create standalone ABI files for all smart contracts')
   .addOptionalParam('silent', 'Provide less details in the output', false, types.boolean)
   .setAction(async (args, hre) => {
     if (!fs.existsSync('./artifacts')) {
-      throw new Error('The directory "artifacts" was not found. Make sure you run "yarn compile" first.');
+      throw new Error('The directory "artifacts" was not found. Make sure you run "pnpm compile" first.');
```

Comment:
@alexanderattar fyi, this package is not really integrated yet. It exists here but is not working.

Reply:
okay that's fine. we can deal with it later. i was just updating references to `yarn`